wiki/index.md

```diff
@@ -0,0 +1,38 @@
+<div id="header_wrap" class="outer">
+
+<header class="inner">
+
+<section id="downloads">
+
+Download this project as a .zip file [REMReM-Semantics.zip
+](https://github.com/eiffel-community/eiffel-remrem-semantics/zipball/master) 
+
+Download this project as a tar.gz file [REMReM-Semantics.tar
+](https://github.com/eiffel-community/eiffel-remrem-semantics/tarball/master)
+
+</header>
+
+</div>
+
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## Introduction
+
+Eiffel REMReM Semantics is a set of protocol defined messages that contains templates and schemas for Eiffel message types. It implements the [Eiffel REMReM Protocol Interface](https://eiffel-community.github.io/eiffel-remrem-protocol-interface/). Eiffel REMReM Semantics is used to check, generate and validate Eiffel messages. This project contains JSON schemas to validate JSON message to met the Eiffel protocol. This project contains information about how to form a routing key that will be added to the generated message.
```

Comment:
It would be good to have a link to further reading from this index page. Perhaps to usage or installation, so the user knows how to get started?

Reply:
Added link to usage and installation